.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env
 
```

Comment:
Works perfectly fine but just a quick tips : 

if those are all located under the same directories - you could ignore them all by doing something like `/env` assuming they live under `root/env`  More pattern here : https://www.atlassian.com/git/tutorials/saving-changes/gitignore